examples/resources/grafana_slo/resource_big_tent.tf

```
@@ -0,0 +1,119 @@
+resource "grafana_slo" "big_tent_test" {
+  name        = "Terraform Testing - Big Tent"
+  description = "Terraform Description - Big Tent"
+  query {
+    freeform {
+      query = jsonencode([
+{
+aggregation: "Sum",
+alias: "",
+application: "57831",
+applicationName: "petclinic",
+datasource: {
+type: "dlopes7-appdynamics-datasource",
+uid: "appdynamics_localdev"
+},
+delimiter: "|",
+isRawQuery: false,
+metric: "Overall Application Performance|Calls per Minute",
+queryType: "metrics",
+refId: "total",
+rollUp: true,
+schemaVersion: "3.9.5",
+transformLegend: "Segments",
+transformLegendText: ""
+},
+{
+aggregation: "Sum",
+alias: "",
+application: "57831",
+applicationName: "petclinic",
+datasource: {
+type: "dlopes7-appdynamics-datasource",
+uid: "appdynamics_localdev"
+},
+intervalMs: 1000,
+maxDataPoints:43200,
+delimiter: "|",
+isRawQuery: false,
+metric: "Overall Application Performance|Calls per Minute",
+queryType: "metrics",
+refId: "also_total",
+rollUp: true,
+schemaVersion: "3.9.5",
+transformLegend: "Segments",
+transformLegendText: ""
+},
+{
+conditions: [
+{
+evaluator: {
+params: [
+0,
+0
+],
+type: "gt"
+},
+operator: {
+type: "and"
+},
+query: {
+params: []
+},
+reducer: {
+params: [],
+type: "avg"
+},
+type: "query"
+}
+],
```

```suggestion
conditions: [
{
evaluator: {
params: [
0,
0
],
type: "gt"
},
operator: {
type: "and"
},
query: {
params: []
},
reducer: {
params: [],
type: "avg"
},
type: "query"
}
],
```
we don't need these part for "math" expressions. We keep copy/pasting it... but we should stop